eslint.config

```diff
@@ -0,0 +1,9 @@
+{
+  "parserOptions": { "ecmaVersion": 8 },
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+  }
+}
```

Comment:
This file should be deleted, as we do not need two config files for our projects, this is the one stickler is listening to, and the one that is linting locally is fundamentally different, so we are bound to have errors, please do refer to [The Microverse Linters Repo](https://github.com/microverseinc/linters-config/tree/master/javascript); and follow the instructions step by step in order to be able to have a proper configuration. The stickler errors you might have with the configuration as it is now are not reflective of the errors you have on your local dev env. So let's make those changes.